src/lib/telldus-accessory.js

```diff
@@ -47,6 +48,12 @@ class TelldusAccessory {
     this.log = string => log(`[${this.name}]: ${string}`)
     this.Characteristic = homebridge.hap.Characteristic
     this.Service        = homebridge.hap.Service
+
+    // Support age limit for sensors to avoid stale information.
+    this.isStale = sensor => {
+      const maxAge = this.data.maxAge || this.config.maxAge || 60 * 60;
```

Comment:
This is *per sensor*, so it can't be global. `this.data.maxAge` comes from a sensor level configuration, for example if you have one sensor that is just out of reach, it can be used to allowing that to check in at a lower rate than the others. (or at a higher rate).
`this.config.maxAge` comes from global config level, and the last is default value if nothing is set. The magic number could of course be set globally.